service-wfs-client/src/main/java/org/oskari/service/wfs/client/CoordinateTransformer.java

```diff
@@ -0,0 +1,77 @@
+package org.oskari.service.wfs.client;
+
+import org.geotools.data.simple.SimpleFeatureCollection;
+import org.geotools.data.simple.SimpleFeatureIterator;
+import org.geotools.feature.DefaultFeatureCollection;
+import org.geotools.feature.simple.SimpleFeatureBuilder;
+import org.geotools.feature.simple.SimpleFeatureTypeBuilder;
+import org.geotools.geometry.jts.JTS;
+import org.geotools.referencing.CRS;
+import org.opengis.feature.simple.SimpleFeature;
+import org.opengis.feature.simple.SimpleFeatureType;
+import org.opengis.geometry.MismatchedDimensionException;
+import org.opengis.referencing.FactoryException;
+import org.opengis.referencing.NoSuchAuthorityCodeException;
+import org.opengis.referencing.crs.CoordinateReferenceSystem;
+import org.opengis.referencing.operation.MathTransform;
+import org.opengis.referencing.operation.TransformException;
+
+import com.vividsolutions.jts.geom.Geometry;
+
+public class CoordinateTransformer {
+
+    private final CoordinateReferenceSystem from;
```

Comment:
Any opinion if we should add these to https://github.com/oskariorg/oskari-server/blob/master/service-map/src/main/java/fi/nls/oskari/map/geometry/ProjectionHelper.java or deprecate the helper or just have these as separate "helpers"?

Reply:
Hard to say, this one is a bit different, since the "user" doesn't have to manage CoordinateReferenceSystems or MathTransforms in their code, but they still get reused.